script.js

```diff
@@ -1,12 +1,156 @@
-//You can edit ALL of the code here
+//Get DOM elements
+const rootHtml = document.getElementById("root"); // Get the root element where we'll add the episode cards
+const searchInputHtml = document.getElementById("search-input");
+const searchResultDisplayHtml = document.querySelector(".search-display");
+const episodeSelectHTML = document.querySelector("#select-html");
+
+// Global variables
+let allEpisodes;
+
+// Fetch episodes function
 function setup() {
```

Comment:
```suggestion
const setup = async () => {
```
I recommend to change simple fetch for async/await

Reply:
Thank you for telling me about async and await. I will try to learn about them and incorporate them in my code!